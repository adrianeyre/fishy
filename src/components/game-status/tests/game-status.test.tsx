import React from 'react';
import { shallow } from 'enzyme';

import GameStatus from '../game-status';
import IGameStatusProps from '../interfaces/game-status-props';

describe('Game Status', () => {
	it('Should render correctly', () => {
		const defaultProps: IGameStatusProps = {
			score: 1000,
			lives: 5,
		};

		const gameStatus = shallow(<GameStatus {...defaultProps} />);
		expect(gameStatus).toMatchSnapshot();
	});
});