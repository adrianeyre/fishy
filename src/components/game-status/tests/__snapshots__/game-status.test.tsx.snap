// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game Status Should render correctly 1`] = `ShallowWrapper {}`;
